src/HTMLElement/elements/HTMLAnchorElement.js

```diff
@@ -0,0 +1,160 @@
+import HTMLElement from '../../HTMLElement';
+import Url from '../../utils/Url';
+
+/**
+ * The HTMLAnchorElement interface represents hyperlink elements and provides special properties and methods
+ * (beyond those of the regular HTMLElement object interface they also have available to them by inheritance)
+ * for manipulating the layout and presentation of such elements.
+ *
+ * @see https://developer.mozilla.org/en/docs/Web/API/HTMLAnchorElement
+ * @class HTMLAnchorElement
+ */
+export default class HTMLAnchorElement extends HTMLElement {
+    constructor() {
+        super();
+        this._href = new Url();
+    }
+    /**
+     * @inheritDoc
+     */
+    setAttribute(key, value) {
+        super.setAttribute(key, value);
```

Comment:
Seems it in external tests i'm using html-document for. I'll add test and check.